cmd/podman/kube/play.go

```diff
@@ -138,6 +138,7 @@ func playFlags(cmd *cobra.Command) {
 	flags.BoolVarP(&playOptions.Quiet, "quiet", "q", false, "Suppress output information when pulling images")
 	flags.BoolVar(&playOptions.TLSVerifyCLI, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
 	flags.BoolVar(&playOptions.StartCLI, "start", true, "Start the pod after creating it")
+	flags.BoolVar(&playOptions.Force, "force", false, "remove volumes")
```

Comment:
Capitalize the R in remove please

Reply:
And can you mention that this only works with `--down`?

Reply:
Finally: can you add a check to error if `--force` is passed but `--down` is not?

Reply:
Fixed